CONTRIBUTING.md

```diff
@@ -47,5 +47,5 @@ Please sign off your commits, to show that you agree to publish your changes und
 , and to indicate agreement with [Developer Certificate of Origin (DCO)](https://developercertificate.org/).
 
 ```shell
-git commit --signed-off ...
```

Comment:
this change appear to be unrelated to the intended feature.
please revert it here and open a dedicated PR for it.

Reply:
fixed via 5af703ae20cd7ce9ff1b3d11b8ed9bea82e71c09